.gitignore

```diff
@@ -0,0 +1,100 @@
+# Dependency directory
+node_modules
+dist/
```

Comment:
If my understanding is right regarded what has been decided, only the `node_modules` should be ignored from the scm, am I right ?